README.md

```diff
@@ -151,6 +151,25 @@ fn main() -> Result<(), bdk::Error> {
 }
 ```
 
+## Testing
+
+### Unit testing
+
+```
+cargo test
+```
+
+### Integration testing
+
+Integration testing require testing features, for example:
+
+```
+cargo test --features test-electrum
```

Comment:
💡  Can the feature flag situation be improved? How about `test-blockchains` + `electrum` does this and we can get rid of `test-electrum`.

Reply:
The problem to do this is the different testing features required by esplora (`electrsd/esplora_a33e97e1`) and electrum (`electrsd/electrs_0_8_10`)

Previously, I even though to remove `test-*` completely (and simply run blockchain electrum test with electrum feature), in this case other than previously described issue there was another issue to include dep like `bitcoincore-rpc` only when testing.